pkg/logtailing/tailer.go

```diff
@@ -190,6 +201,15 @@ func (tailer *Tailer) processRequestLogEvent(msg websocket.IncomingMessage) {
 	color := ansi.Color(os.Stdout)
 	outputStr := fmt.Sprintf("%s [%d] %s %s [%s]", color.Faint(localTime), coloredStatus, payload.Method, payload.URL, requestLink)
 	fmt.Println(outputStr)
+
+	var payloadError RedactedError
+	if err := json.Unmarshal([]byte(payload.Error), &payloadError); err != nil {
```

Comment:
Assuming the stripecli service sends the `error` field as a hash, not as a string, we don't need to unmarshal it ourselves. Just unmarshaling the top-level `EventPayload` will do the right thing.

Reply:
Oh cool, I didn't know it would work recursively like that, nice!